src/VirtoCommerce.CatalogModule.Core/Model/DynamicAssociations/DynamicAssociationValue.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections.Generic;
+
+namespace VirtoCommerce.CatalogModule.Core.Model.DynamicAssociations
+{
+    public class DynamicAssociationValue
```

Comment:
I'd rename this somehow to have an idea of the purpose of this class, e.g. DynamicAssociationCondition